less/pageLevelProgressIndicator.less

```diff
@@ -2,7 +2,7 @@
 // --------------------------------------------------
 .pagelevelprogress {
   &__indicator {
-    display: inline-block;
+    display: flex;
     vertical-align: middle;
```

Comment:
```suggestion
```
vertical-align has no effect on this element since it’s not an inline or table-cell element.